glossary.md

```diff
@@ -1,16 +1,80 @@
 You may also need to check https://chromium.googlesource.com/chromiumos/docs/+/master/glossary.md.
 
-* LGTM: "Looks good to me", commonly used to approve a code review.
-* PTAL: Please take a look.
-* RSLGTM: "Rubber-stamp looks good to me". The reviewer approving without doing
+* ABI: [Application Binary Interface](https://en.wikipedia.org/wiki/Application_binary_interface)
+* ASAP: As Soon As Possible.
+* ASYNC: Asynchronous
+* BE: Big Endian - Byte order style in a multibyte data.
+  (Opposite of LE, Little Endian)
+* CI: Continuous Integration.
+* CITGM: Canary In The Gold Mine - a smoke test unit in the repo that
+  contain popular npm modules.
+* CJS: Common JS.
+* CLDR: Common Locale Data Repository.
```

Comment:
This does not seem to be useful for people who don't know what CLDR is

Reply:
Thanks, linked it to Wikipedia explanation.